specification/ml/put_job/MlPutJobRequest.ts

```diff
@@ -44,6 +44,9 @@ export interface Request extends RequestBase {
      */
     job_id: Id
   }
+  query_parameters: {
+    ignore_throttled?: boolean
+  }
```

Comment:
> The only thins missing server-side proof is `ignore_throttled` as query parameter in `MlPutJobRequest`, I couldn't find it in parent classes either, so I'm trusting the validation on that one.

This took me a while to figure out! `ignore_throttled` is part of IndicesOptions. When passed as a query parameter, it's allowed directly, which is what `IndicesOptions.fromRequest` does here: https://github.com/elastic/elasticsearch/blob/64e0902f58a350b666708a85c9506dccdd7ecc0b/x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/job/RestPutJobAction.java#L47.

This means we should add all of them, as done in `update_datafeed` for example: https://github.com/elastic/elasticsearch-specification/blob/d9bf86fe712f25f7ef5c0037eab484a22008c896/specification/ml/update_datafeed/MlUpdateDatafeedRequest.ts#L53-L81